flink-runtime/src/test/java/org/apache/flink/runtime/filecache/FileCacheDirectoriesTest.java

```diff
@@ -164,16 +165,13 @@ public void testDirectoryCleanUp() throws Exception {
 		assertTrue(fs.exists(cacheFile));
 
 		fileCache.releaseJob(jobID, attemptID2);
-		// still should be available, file will be deleted after 5 seconds
+		// still should be available, file will be deleted after 200 milliseconds
 		assertTrue(fileStatus.isDir());
 		assertTrue(fs.exists(cacheFile));
```

Comment:
what you _could_ also do is pass in a `ScheduledExecutorService`. You could then intercept the delete process, verify the interval arguments and fire the process at your leisure.

I'm worried that in it's current form the test is either not stable or takes longer than it should.